autoemulate/core/model_selection.py

```diff
@@ -222,11 +231,17 @@ def bootstrap(
 
         # Compute metrics for this bootstrap sample
         for metric in metrics:
+            kwargs = metric_kwargs.get(metric.name, {})
             metric_scores[metric.name][i] = evaluate(
-                y_pred, y_bootstrap, metric=metric, n_samples=n_samples
+                y_pred,
+                y_bootstrap,
+                metric=metric,
+                n_samples=n_samples,
+                metric_kwargs=kwargs,
             )
```

Comment:
I think if going with this approach - I think this might need to be added above for the `n_bootstraps=None` case?